t/t2241-queue-cmd-list.t

```diff
@@ -0,0 +1,371 @@
+	       -o "{defaults.timelimit},{limits.timelimit},{defaults.timelimit!F},{limits.timelimit!F}" \
+	       --from-stdin < policy_defaults_config.in > policy_defaults_timelimit.out &&
+       echo "3600.0,14400.0,1h,4h" > policy_defaults_timelimit.exp &&
+       test_cmp policy_defaults_timelimit.exp policy_defaults_timelimit.out
+'
+test_expect_success 'flux-queue: defaults config ranges' '
+       flux queue list -n \
+	       -o "{limits.range.nnodes},{limits.range.ncores},{limits.range.ngpus}" \
+	       --from-stdin < policy_defaults_config.in > policy_defaults_range.out &&
+       echo "2-10,1-inf,1-inf" > policy_defaults_range.exp &&
+       test_cmp policy_defaults_range.exp policy_defaults_range.out
+'
+test_expect_success 'flux-queue: defaults config mins' '
+       flux queue list -n \
+	       -o "{limits.min.nnodes},{limits.min.ncores},{limits.min.ngpus}" \
+	       --from-stdin < policy_defaults_config.in > policy_defaults_min.out &&
+       echo "2,1,1" > policy_defaults_min.exp &&
+       test_cmp policy_defaults_min.exp policy_defaults_min.out
+'
+test_expect_success 'flux-queue: defaults config maxes' '
+       flux queue list -n \
+	       -o "{limits.max.nnodes},{limits.max.ncores},{limits.max.ngpus}" \
+	       --from-stdin < policy_defaults_config.in > policy_defaults_max.out &&
+       echo "10,inf,inf" > policy_defaults_max.exp &&
+       test_cmp policy_defaults_max.exp policy_defaults_max.out
+'
+
+#
+# named queues
+#
+
+test_expect_success 'create several named queues without default' '
+	cat <<-EOF > queues_no_default.in
+{
+  "queues": {
+    "batch": {},
+    "debug": {}
+  }
+}
+EOF
+'
+# we can't predict order of listing, so we grep and count lines
+test_expect_success 'flux-queue: queue config, no default marked' '
+	flux queue list -n \
+		-o "{queue},{queuem}" \
+		--from-stdin < queues_no_default.in > queues_no_default.out &&
+	test $(grep "^batch,batch$" queues_no_default.out | wc -l) -eq 1 &&
+	test $(grep ^"debug,debug$" queues_no_default.out | wc -l) -eq 1
+'
+
+# Configure
+#
+# [policy.jobspec.defaults.system]
+# queue = "batch"
+# duration = "1h"
+#
+# [policy.limits]
+# duration = "4h"
+# job-size.min.nnodes = 2
+# job-size.max.nnodes = 10
+# job-size.min.ncores = 1
+# job-size.max.ncores = -1
+# job-size.min.ngpus = 1
+# # job-size.max.ngpus left out to test default of infinity
+#
+# [queues.batch]
+# policy.jobspec.defaults.system.duration = "2h"
+#
+# [queues.batch.policy.limits]
+# duration = "8h"
+# job-size.min.nnodes = 5
+# job-size.max.nnodes = 15
+# job-size.min.ncores = 1
+# job-size.max.ncores = 20
+# job-size.min.ngpus = 1
+# job-size.max.ngpus = 4
+#
+# [queues.debug.policy.limits]
+# duration = "2h"
+# job-size.max.nnodes = 4
+# job-size.min.ncores = 2
+#
+# N.B. Note that many values in queues.debug.policy.limits left out to
+# test that system wide config is parsed correctly
+test_expect_success 'create queue defaults config for tests' '
+	cat <<-EOF > queues_config.in
+{
```

Comment:
Wow, this makes me wonder if it would be easier to use a TOML config (in comments above) along with `flux config load` and let `flux queue list` read config from the broker?

Reply:
Yeah, i initially tried to use a linux command line toml->json converter, but didn't know if such a tool could safely be assumed to be installed.

To me, its under that threshold of annoyance to support reading config in toml format, like adding a `--from-stdin-toml` or whatever.  Or I guess I could try and detect toml vs json.

Reply:
Oh, I just meant that the broker reads toml then returns the config in json, so why not use it since the test is running under Flux anyway (i.e. do not use `--from-stdin` at all). However, I may have missed what the test is doing.

Reply:
ohhh just load it via toml normally.  yeah, i guess that would be cleaner :-)